games-mud/tkmoo/tkmoo-0.3.32-r1.ebuild

```diff
@@ -1,44 +1,45 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit eutils
+EAPI=8
 
-MY_PN=${PN/moo/MOO-light}
-MY_P=${P/moo/MOO-light}
+inherit desktop
+
+MY_PN="${PN/moo/MOO-light}"
+MY_P="${P/moo/MOO-light}"
 DESCRIPTION="MOO Client written in Tcl/Tk"
 HOMEPAGE="http://www.awns.com/tkMOO-light/"
 SRC_URI="http://www.awns.com/tkMOO-light/Source/${MY_P}.tar.gz"
 
 LICENSE="tkMOO"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
 DEPEND="
-	>=dev-lang/tcl-8.3.3:0=
-	>=dev-lang/tk-8.3.3:0="
-RDEPEND=${DEPEND}
+	dev-lang/tcl:0=
+	dev-lang/tk:0="
+RDEPEND="${DEPEND}"
 
-S=${WORKDIR}/${MY_P}
+S="${WORKDIR}/${MY_P}"
 
 PATCHES=(
 	"${FILESDIR}/${PV}-Makefile-noclean.patch"
-		"${FILESDIR}/${PV}-keys-workaround.patch"
+	"${FILESDIR}/${PV}-keys-workaround.patch"
 )
 
 src_compile() {
 	emake \
 		WISH="$(type -P wish)" \
-		TKMOO_LIB_DIR="/usr/$(get_libdir)/${MY_PN}" \
-		TKMOO_BIN_DIR=/usr/bin
+		TKMOO_LIB_DIR="${EPREFIX}/usr/$(get_libdir)/${MY_PN}" \
+		TKMOO_BIN_DIR="${EPREFIX}/usr/bin"
```

Comment:
I personally came to prefer replacing those backslash `\` mess by a variable even if it's just for 2-3 lines. i.e.
```
local emakeargs=(
	VAR=val
)
emake "${emakeargs[@]}"
```
`\` is also more error prone, if you forget a backlash it doesn't necessarily fail and bash just sets the value in the environment.